### ISSUE TITLE
Update global link color to be accessible

### ISSUE THREAD

**Issue by jrjohnson** [ilios/frontend#4841]
Our current link blue `#009ccc` does not have enough contrast with the text color and background color to be accessible. We'll need to explore other color options for links along with some other accommodations like underlining all links to meet a11y requirements.

**Comment by jrjohnson** [comment]
Team Discussion:
We'll try some different blues, also underline on focus/ hover.

**Comment by jrjohnson** [comment]
Fixed in https://github.com/ilios/common/pull/882
